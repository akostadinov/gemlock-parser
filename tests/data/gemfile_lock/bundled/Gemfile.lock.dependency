GEM
  remote: https://rubygems.org/
  specs:
    ast (2.4.2)

PLATFORMS
  ruby

DEPENDENCIES
  ast
  bundler (= 2.0.1)

BUNDLED WITH
   2.0.1
