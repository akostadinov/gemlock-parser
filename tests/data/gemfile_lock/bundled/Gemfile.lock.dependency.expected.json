[
  {
    "name": "ast",
    "version": "2.4.2",
    "platform": "ruby",
    "pinned": false,
    "remote": "https://rubygems.org/",
    "type": "GEM",
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [],
    "dependencies": {
      "ast@2.4.2": {}
    }
  },
  {
    "name": "bundler",
    "version": "2.0.1",
    "platform": null,
    "pinned": false,
    "remote": null,
    "type": null,
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [
      "= 2.0.1"
    ],
    "dependencies": {
      "bundler@2.0.1": {}
    }
  }
]
